### PR TITLE
Added Captain Of Industry

### PR DESCRIPTION
**Pull request by copypasta1**

### CODE REVIEW

**Comment by dcbroad3** [comment]
Nice!

I had started working on a data set of my own, but I may let this merge first and then just make any necessary changes to the data in my pass. I'll try to give it a thorough look soon.

One thing: I generally try to use three-letter-acronyms in the data sets, I was intending to use `coi` for Captain of Industry, it looks like you set it up as `capt`. The dedicated route doesn't matter to me as much as the folder name does. The angular route could be `coi`, `capt`, or even `captains` but I think I'd like the folder name to be `coi` (and the internal abbreviation, like `TITLE_COI` not `TITLE_CAPT`).

**Comment by dcbroad3** [comment]
Another comment: Go ahead and increment the revision number in the version in `package.json`.

**Comment by copypasta1** [comment]
> Nice!
> 
> I had started working on a data set of my own, but I may let this merge first and then just make any necessary changes to the data in my pass. I'll try to give it a thorough look soon.
> 
> One thing: I generally try to use three-letter-acronyms in the data sets, I was intending to use `coi` for Captain of Industry, it looks like you set it up as `capt`. The dedicated route doesn't matter to me as much as the folder name does. The angular route could be `coi`, `capt`, or even `captains` but I think I'd like the folder name to be `coi` (and the internal abbreviation, like `TITLE_COI` not `TITLE_CAPT`).

I will work on changing these. I had no preference so I will get these changed.
I will also work to clear up the failed tests.... I didnt look at them at all before now

**Comment by dcbroad3** [comment]
Redeploying to https://factoriolab.github.io/staging using PR https://github.com/factoriolab/factoriolab/pull/787

File: src/app/data/index.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import { AppData, Game, Mod } from '~/models';
```
Comment (dcbroad3) [inline src/app/data/index.ts:1]: This file looks like it was accidentally added as a duplicate

File: src/app/routes/list/list.component.html [hunk 2]
```diff
@@ -393,7 +396,7 @@
                     }}</span>
                   </div>
                 </td>
-                <td class="recipe-icon split-column">
+                <td class="recipe-icon split-column testclass">
```
[the remark this answers is not in the document]
Comment (copypasta1) [reply src/app/routes/list/list.component.html:399]: Right, that was when I was trying to figure out Icons for the maintenance column

File: src/app/data/index.ts (hunk 1 above)
Comment (copypasta1) [reply src/app/data/index.ts:1]: Yes, duplicate

**Pull request merged**